advanced-usage/future-mocking.md

```diff
@@ -1,619 +1,513 @@
-            Mock.Arrange(() => serviceMock.Save(valueToSave)).Raises(() => serviceMock.Saved += null, new EntrySavedEventArgs(valueToSave));
-        };
-    
-        private Because of = () =>
-        {
-            sut = new Form();
-            sut.SaveToDatabase(valueToSave);
-        };
-    
-        private It should_assert_that_label_contains_expected_valueToSave = () =>
-            sut.label1.Text.ShouldEqual("Saved string : " + valueToSave);
-    
-        static Form sut;
-        const string valueToSave = "Raise Event";
-    }
-  {{endregion}}
-
-  #### __[VB]__
-
-  {{region FutureMocking#SampleCode6}}
-    <Subject(GetType(Form))>
-    Public Class when_save_to_database_is_invoked_on_form
-        Private context As Establish = Function()
-                                          Dim serviceMock As IEntryService = Mock.Create(Of EntryService)()
-                                          Mock.Arrange(Function() serviceMock.Save(valueToSave)).Raises(Function() CSharpImpl.__Assign(serviceMock.Saved, Nothing), New EntrySavedEventArgs(valueToSave))
-                                      End Function
-
-        Private [of] As Because = Function()
-                                      sut = New Form()
-                                      sut.SaveToDatabase(valueToSave)
-                                  End Function
-
-        Private should_assert_that_label_contains_expected_valueToSave As It = Function() sut.label1.Text.ShouldEqual("Saved string : " & valueToSave)
-        Shared sut As Form
-        Const valueToSave As String = "Raise Event"
-
-        Private Class CSharpImpl
-            <Obsolete("Please refactor calling code to use normal Visual Basic assignment")>
-            Shared Function __Assign(Of T)(ByRef target As T, value As T) As T
-                target = value
-                Return value
-            End Function
-        End Class
-    End Class
-  {{endregion}}
-
-Here we can see that although no instance is supplied to the target UI class, JustMock picks up the intended setup from the context.
-
-## See Also
-
- * [Fluent Mocking]({%slug justmock/basic-usage/fluent-mocking%})
+---
+title: Working with Instances
+page_title: Working with Instances | JustMock Documentation
+description: Working with Instances
+previous_url: /advanced-usage-future-mocking.html
+slug: justmock/advanced-usage/future-mocking
+tags: future,mocking, mock, instance, ignore, create
+published: True
+position: 8
+---
+
+# Working With Instances of Mocked Objects
+
+With the advanced features of __Telerik® JustMock__ you can apply the arrangements you need and configure them to execute for **each instance** no matter where and when it is being created in the current context. With this feature, you don't need to arrange every instance explicitly. You will find this handy, especially in cases when you have to mock third party controls and tools where you have little control over how they are created.
```

Comment:
When searching in google for future mocking the first 160 symbols from this text will be shown meaning that the first 160 symbols must contain the most valuable information. Having that in mind please simplify the beginning of the description and rework it to contain the most valuable text.